compose.env

```diff
@@ -32,8 +32,6 @@ OLLAMA_MODEL=qwen2:1.5b
 ## INFINITY VARS
 INFINITY_EMBEDDING_MODEL=mixedbread-ai/mxbai-embed-large-v1
 INFINITY_RERANKING_MODEL=mixedbread-ai/mxbai-rerank-xsmall-v1
-## INFINITY_API_KEY, only required if you enable API KEY auth on infinity container
-INFINITY_API_KEY=
```

Comment:
Any problems in keeping this with empty value?

Reply:
works we can keep an empty API key.